plugins/Widgetize/lang/en.json

```diff
@@ -4,6 +4,19 @@
         "PluginDescription": "Display any Matomo report in your website or app with a simple \"embed\" HTML tag.",
         "ViewAccessRequired": "This user has at least some write access. Only tokens of users who have only view access can be used. Check %1$s for more info.",
         "TopLinkTooltip": "Export Matomo reports as widgets and embed the dashboard in your app as an iframe.",
-        "TooHighAccessLevel": "This user has super-user access. Embedding widgets with super-user token authentication is not allowed. %1$sCheck the FAQ for more info.%2$s"
+        "TooHighAccessLevel": "This user has super-user access. Embedding widgets with super-user token authentication is not allowed. %1$sCheck the FAQ for more info.%2$s",
+        "SelectAReport": "Select a report, and copy paste in your page the embed code below the widget:",
+        "Reports": "Widgetize reports",
+        "Intro": "With Matomo, you can %s on your blog, website, or intranet dashboard... in one click.",
+        "Intro2": "export your Web Analytics reports",
+        "DisplayDashboardInIframe": "You can also display the full Matomo dashboard in your application or website in an IFRAME (%s).The date parameter can be set to a specific calendar date, \"today\", or \"yesterday\". The\n period parameter can be set to \"day\", \"week\", \"month\", or\n \"year\". The language parameter can be set to the language code of a translation, such as language=fr. For example, for idSite=1 and date=yesterday, you can write: <br/>",
+        "DisplayDashboardInIframe2": "see example",
+        "DisplayDashboardInIframeAllSites": "You can also widgetize the all websites dashboard in an IFRAME (%s)<br/>",
+        "DisplayDashboardInIframeAllSites2": "see example",
+        "ViewableAnonymously": "If you want your widgets to be viewable by everybody, you first have to set the 'view' permissions to the anonymous user in the %s. <br />Alternatively, if you are publishing widgets on a password protected or private page, you don't necessarily have to allow 'anonymous' to view your reports. In this case, you can add the secret <code>token_auth</code> parameter in the widget URL. You can manage your auth tokens on your %s.",
+        "ViewableAnonymously2": "Users Management section",
+        "ViewableAnonymously3": "Security page",
```

Comment:
Same here. Might be better to combine them again.
Also if there is more than one placeholders we need to use them numbered (e.g. `%1$s`, `%2$s`, ....) as in some languages the order might need to be changed, which won't work otherwise.